src/osd/osd_types.cc

```diff
@@ -4166,6 +4167,9 @@ void pg_log_t::decode(bufferlist::const_iterator &bl, int64_t pool)
   if (struct_v >= 5)
     decode(can_rollback_to, bl);
 
+  if (struct_v >= 8)
+    decode(rollback_invalidate, bl);
+
```

Comment:
...and put this at teh end too, along with an else clause to set this value to whatever new code should see when getting a message/structure from old code.